### PR TITLE
Issue fix

### PR DESCRIPTION
**Pull request by Programmer-yar**
Changes can be viewed at:
http://localhost:8000/
http://localhost:8000/polls/task-flow/

Include changes for https://github.com/turian/audio-discrimination-crowdsource/issues/14, https://github.com/turian/audio-discrimination-crowdsource/issues/15, https://github.com/turian/audio-discrimination-crowdsource/issues/17, https://github.com/turian/audio-discrimination-crowdsource/issues/19, https://github.com/turian/audio-discrimination-crowdsource/issues/20

Screenshot from 2022-09-14 18-09-17
Screenshot from 2022-09-14 18-07-24

### CODE REVIEW

File: django_app/settings.py [hunk 1]
```diff
@@ -110,7 +110,6 @@
         },
         # to capture email/profile data from google Oauth
         "SCOPE": [
-            "profile",
```
Comment (turian) [inline django_app/settings.py:113]: Again. Why don't you break this stuff into separate PRs that do separate things?


If you submitted this one-liner as a PR I'd approve it immediately. But the rest of it needs some changes. Why not make multiple independent PRs like we've talked about several times?

File: themeapp/templates/widgets/top-bar.html [hunk 2]
```diff
@@ -10,7 +10,6 @@
                 <li class="nav-item"><a class="nav-link" href="{% url 'account_logout' %}">Log Out</a></li>
                 {% else %}
                 <li class="nav-item"><a class="nav-link" href="{% url 'account_login' %}">Log In</a></li>
-                <li class="nav-item"><a class="nav-link" href="{% url 'account_signup' %}">Sign Up</a></li>
                 {% endif %}
```
Comment (turian) [inline themeapp/templates/widgets/top-bar.html:13]: Again. Separate PR for separate functionality. I would merge this change immediately.

File: polls/views.py [hunk 3]
```diff
@@ -25,12 +25,12 @@ def get(self, request):
         }
         return render(request, self.template_name, context)
 
-class TaskFlowView(LoginRequiredMixin, UserPassesTestMixin, View):
+class TaskFlowView(LoginRequiredMixin, View):
```
Comment (turian) [inline polls/views.py:28]: Why are you removing the mixin? Maybe you need a new mixin?

File: polls/views.py [hunk 4]
```diff
@@ -25,12 +25,12 @@ def get(self, request):
...
     template_name = "polls/task_flow.html"
 
     def get(self, request):
         can_continue, should_rest, _ = check_user_work_permission(request.user)
-        if should_rest:
+        if should_rest or request.user.is_locked:
```
Comment (turian) [inline polls/views.py:33]: Uh this is wrong and badly designed.

a) If the user should rest, they don't go to the auth-flow. They get a page saying: "Thank you for your work come back in X minutes." or whatever I put in my spec. If you don't want to implement that yet, then you put a `#TODO: comment in the code`

b) You're doing an if check on `request.user.is_locked`. Those are going to be littered everywhere in the code and you're going to forget to add them somewhere. Why not just make a mixin or something that makes it really obvious which views / apps / whatever you can access if you are not frozen. And if you are frozen, you get a separate "Thank you for your work" page (or whatever I put in the spec, please read it and dont make me dig it up and repeat myself.) And if that's not ready then you put a `#TODO` there.

File: django_app/settings.py (hunk 1 above)
Comment (Programmer-yar) [reply django_app/settings.py:113]: Ok,
From now on I will make PRs further small

File: polls/views.py (hunk 3 above)
Comment (Programmer-yar) [reply polls/views.py:28]: This is builtin mixin and is used to direct to 403 only in case test fails

File: polls/views.py (hunk 4 above)
Comment (Programmer-yar) [reply polls/views.py:33]: a) That's already added to auth flow and I just didn't want to repeat it. I have implemented that already and I want to work on it.
b) You mean some decorator or mixin, if that's the case still we will have to add that custom decorator on all views. I will check for decorator or mixin.

Comment (turian) [reply polls/views.py:33]: @Programmer-yar As discussed in telegram:

MAKE an issue to fix it, add a todo in the code, reply with a comment saying: I'll fix it in this issue.

Then we are DONE.

You can't go down endless rabbit holes. You need to break up the work into tiny bite sized pieces. Creating an issue to improve something later is a FAST way to be DONE.

Comment (Programmer-yar) [reply polls/views.py:34]: I will fix in this issue https://github.com/turian/audio-discrimination-crowdsource/issues/33
